src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -494,13 +494,27 @@ public function previousAll()
     }
 
     /**
-     * Returns the parents nodes of the current selection.
+     * Returns the parent nodes of the current selection.
      *
      * @return static
      *
      * @throws \InvalidArgumentException When current node is empty
      */
     public function parents()
+    {
+        @trigger_deprecation('symfony/dom-crawler', '5.3', sprintf('The %s() method is deprecated, use ancestors() instead.', __METHOD__));
```

Comment:
AFAIK the `@` isnt needed

Reply:
fixed in https://github.com/symfony/symfony/commit/e970027682e27b2605996e6ae3c082dde2a8d828
good catch, thank you!

Reply:
Thank you @chalasr!